docs/rule/no-unnecessary-curly-braces-for-strings.md

```diff
@@ -0,0 +1,21 @@
+# no-unnecessary-curly-braces-for-strings
```

Comment:
Perhaps just `no-unnecessary-curly-strings` for the rule name? Would align nicely with this other new rule:

* #2792 (likely being renamed to `no-unnecessary-curly-parens`)

Reply:
not able to change the rule names and run the tests..should i raise a new PR? getting this logs followed by errors
<img width="769" alt="Screenshot 2023-02-06 at 1 11 20 AM" src="https://user-images.githubusercontent.com/59643926/216841227-7269b77a-fd6a-4e37-9ed4-524bf6673fbc.png">

Reply:
No need for a new PR. You just have to make sure all the files and references are renamed.

Reply:
done with the changes @bmish , thank you!